bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/handler/OpenWebNetBridgeHandler.java

```diff
@@ -314,13 +314,27 @@ protected void registerDevice(String ownId, OpenWebNetThingHandler thingHandler)
     /**
      * Un-register a device from this bridge handler
      *
-     * @param oId the device OpenWebNet id
+     * @param ownId the device OpenWebNet id
      */
-    protected void unregisterDevice(String oId) {
-        if (registeredDevices.remove(oId) != null) {
-            logger.debug("un-registered device ownId={}", oId);
+    protected void unregisterDevice(String ownId) {
+        if (registeredDevices.remove(ownId) != null) {
+            logger.debug("un-registered device ownId={}", ownId);
         } else {
-            logger.warn("could not un-register ownId={} (not found)", oId);
+            logger.warn("could not un-register ownId={} (not found)", ownId);
+        }
+    }
+
+    /**
+     * Get an already registered device on this bridge handler
+     *
+     * @param ownId the device OpenWebNet id
+     * @return the registered device Thing handler or null if the id cannot be found
+     */
+    public @Nullable OpenWebNetThingHandler getRegisteredDevice(String ownId) {
+        if (registeredDevices.containsKey(ownId)) {
+            return registeredDevices.get(ownId);
+        } else {
+            return null;
         }
```

Comment:
`Map.get()` returns null if the key doesn't exist. You could simply do:
```suggestion
        return registeredDevices.get(ownId);
```

Reply:
fixed. 
Could you also set this PR label (bug) so it gets tracked in the changelog? (apparently I cannot set label to my PR request), thanks.